src/components/calcite-tile-select/calcite-tile-select.stories.ts

```diff
@@ -1,76 +1,1880 @@
+        knobs: [
+          { name: "icon", value: icon },
+          { name: "heading", value: heading },
+          { name: "description", value: description },
+          { name: "width", value: "full" }
+        ]
+      })
+    )
+    .executeScript(toggleCentered)
+    .snapshot("Icon + Heading + Description")
+
+    // Width Full Heading + Description
+    .executeScript(
+      setKnobs({
+        story: "components-tiles-tile-select--default",
+        knobs: [
+          { name: "heading", value: heading },
+          { name: "description", value: description },
+          { name: "width", value: "full" }
+        ]
+      })
+    )
+    .executeScript(toggleCentered)
+    .snapshot("Heading + Description")
+
+    // Width Full Input Enabled Icon
+    .executeScript(
+      setKnobs({
+        story: "components-tiles-tile-select--default",
+        knobs: [
+          { name: "icon", value: icon },
+          { name: "input-enabled", value: "true" },
+          { name: "width", value: "full" }
+        ]
+      })
+    )
+    .executeScript(toggleCentered)
+    .snapshot("Input Enabled Icon")
+
+    // Width Full Input Enabled Heading
+    .executeScript(
+      setKnobs({
+        story: "components-tiles-tile-select--default",
+        knobs: [
+          { name: "heading", value: heading },
+          { name: "input-enabled", value: "true" },
+          { name: "width", value: "full" }
+        ]
+      })
+    )
+    .executeScript(toggleCentered)
+    .snapshot("Input Enabled Heading")
+
+    // Width Full Input Enabled Description
+    .executeScript(
+      setKnobs({
+        story: "components-tiles-tile-select--default",
+        knobs: [
+          { name: "description", value: description },
+          { name: "input-enabled", value: "true" },
+          { name: "width", value: "full" }
+        ]
+      })
+    )
+    .executeScript(toggleCentered)
+    .snapshot("Input Enabled Description")
+
+    // Width Full Input Enabled Icon + Heading
+    .executeScript(
+      setKnobs({
+        story: "components-tiles-tile-select--default",
+        knobs: [
+          { name: "heading", value: heading },
+          { name: "icon", value: icon },
+          { name: "input-enabled", value: "true" },
+          { name: "width", value: "full" }
+        ]
+      })
+    )
+    .executeScript(toggleCentered)
+    .snapshot("Input Enabled Icon + Heading")
+
+    // Width Full Input Enabled Icon + Description
+    .executeScript(
+      setKnobs({
+        story: "components-tiles-tile-select--default",
+        knobs: [
+          { name: "description", value: description },
+          { name: "icon", value: icon },
+          { name: "input-enabled", value: "true" },
+          { name: "width", value: "full" }
+        ]
+      })
+    )
+    .executeScript(toggleCentered)
+    .snapshot("Input Enabled Icon + Description")
+
+    // Width Full Input Enabled Icon + Heading + Description
+    .executeScript(
+      setKnobs({
```

Comment:
This is most impressive! Great job adding all of these specific tests. 🧪

Reply:
Thank ya sir!  Definitely a lot of work, but well worth it.  We shouldn't miss any more visual regressions with this component any more :)